Release logzio monitoring v7.3.1

ralongit opened the pull request:
## Description 
- Upgrade `logzio-k8s-telemetry` chart to `5.2.1`
  - Fix Pod disruption budget selector label for SPM collector (Contributed by @jod972)
  - Add PriorityClassName for SPM collector (Contributed by @jod972)
  - Add Replica Count for traces & SPM collectors
- Upgrade workflows dependencies ( [#596](https://github.com/logzio/logzio-helm/issues/596) [#597](https://github.com/logzio/logzio-helm/issues/597) [#580](https://github.com/logzio/logzio-helm/issues/580))
- Upgrade `logzio-logs-collector` chart to `2.1.0`
  - **Breaking changes**
    - Upgrade `otel/opentelemetry-collector-contrib` image to version `0.127.0`
      - The `time` attribute is no longer present as field in the log record. The log timestamp is now only available under `@timestamp` in Logz.io.  
  
<!-- 
    "This PR [adds/removes/fixes/replaces] the [feature/bug/etc].."
    Do not leave this blank.
    If relevant, describe the previous behaviour compared to the behaviour post your code change.
-->

## What type of PR is this?
#### (check all applicable)
- [x] 🍕 Feature 
- [x] 🐛 Bug Fix
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [x] 🤖 Build / CI
- [ ] ⏩ Revert

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help from somebody

ralongit committed:
Release `logzio-monitoring` Helm chart v7.3.1

- Upgrade `logzio-k8s-telemetry` chart to `5.2.1`
  - Fix Pod disruption budget selector label for SPM collector (Contributed by @jod972)
  - Add PriorityClassName for SPM collector (Contributed by @jod972)
  - Add Replica Count for traces & SPM collectors

ralongit committed:
Upgrade workflows dependencies

#596 #597 #580

8naama commented on .github/workflows/auto-release-kube-tools.yaml:
```diff
@@ -27,7 +27,7 @@ jobs:
           password: ${{ secrets.DOCKER_PASSWORD }}
       - name: Build and push
-        uses: docker/build-push-action@v5
+        uses: docker/build-push-action@v6
```
upgrading to [v6](https://github.com/docker/build-push-action/releases/tag/v6.0.0) requires to [disbale the docker build summary](https://docs.docker.com/build/ci/github-actions/build-summary/#disable-job-summary) (unless we want to keep it enabled)

8naama reviewed (commented):
I think can also upgrade the logs chart to `2.1.0` on the way

ralongit committed:
Disable kube-tools auto release docker build summary

ralongit committed:
Update supported kubernetes cluster version for EKS

ralongit committed:
Upgrade `logzio-logs-collector` chart to `2.1.0`

  - **Breaking changes**
    - Upgrade `otel/opentelemetry-collector-contrib` image to version `0.127.0`
      - The `time` attribute is no longer present as field in the log record. The log timestamp is now only available under `@timestamp` in Logz.io.
